src/phoenix/auth.py

```diff
@@ -87,7 +87,7 @@ def _set_token_cookie(
     response.set_cookie(
         key=cookie_name,
         value=token,
-        secure=True,
+        # secure=True,
```

Comment:
Can you switch this to a config env var so that people can configure?

Reply:
done